src/layout/CommunitySection/CommunitySection.svelte

```diff
@@ -5,6 +5,8 @@
 	import { Button } from "fluent-svelte";
 	import Profile from "@fluentui/svg-icons/icons/person_32_filled.svg?raw";
 
+	console.log(`data:image/svg+xml;${Profile}`);
```

Comment:
I think this was not intentional.

Reply:
That indeed wasn't, i was just testing something.